src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1215,10 +1215,13 @@
     <value>Do not initialize unnecessarily</value>
   </data>
   <data name="DoNotInitializeUnnecessarilyDescription" xml:space="preserve">
-    <value>The common language runtime initializes all fields to their default values before running the constructor. In most cases, initializing a field to its default value in a constructor is redundant, which degrades performance and adds to maintenance costs. One case where it is not redundant occurs when the constructor calls another constructor of the same class or a base class constructor and that constructor initializes the field to a non-default value. In this case, changing the value of the field back to its default value can be appropriate.</value>
+    <value>The common language runtime initializes all fields to their default values before running the constructor. In most cases, initializing a field to its default value in a constructor is redundant, which adds to maintenance costs and may degrade performance (such as with increased assembly size). One case where it is not redundant occurs when the constructor calls another constructor of the same class or a base class constructor and that constructor initializes the field to a non-default value. In this case, changing the value of the field back to its default value can be appropriate.</value>
   </data>
   <data name="DoNotInitializeUnnecessarilyMessage" xml:space="preserve">
-    <value>Do not initialize unnecessarily</value>
+    <value>Member '{0}' is explicitly initialized to its default value.</value>
+  </data>
+  <data name="DoNotInitializeUnnecessarilyFix" xml:space="preserve">
+    <value>Remove the unnecessary assignment.</value>
```

Comment:
I might be mistaken but I think we decided not to have trailing period for code fix titles, am I right @mavasani ?

Reply:
As it's currently inconsistent, I'm going to go ahead and merge as-is.  If the right answer is to remove the periods, we should have a cleanup PR that removes them from all of the messages consistently.